docs/user/basics/documents-creator.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Documents Creator
+description: Create user-friendly documents
+lang: en
+---
+
+# Documents Creator
+
+The Documents creator converts your documentation data into user-friendly documents. This allows you to reach a diverse audience - from tech-savvy stakeholders to the less digitally fluent decision-makers.
+
+The documents can serve as a documentation basis for audits, conformity checks and customer reporting. Documents are available in the main navigation bar.
```

Comment:
Original:
Documents are available in the main navigation bar.
Proposal:
The Documents Creator is available in the submenu of "Reports" in the main navigation bar